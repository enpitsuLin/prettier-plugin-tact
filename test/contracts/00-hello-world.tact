contract HelloWorld {

    init() {}

    receive() {}

    get fun greeting(): String {
        return "hello world";
    }
}