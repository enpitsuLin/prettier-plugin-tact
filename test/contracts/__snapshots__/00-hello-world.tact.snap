contract HelloWorld {

    receive({})

    get fun greeting(): String {
        return "hello world";
    }
}